gulpfile.ts

```diff
@@ -129,11 +128,6 @@ async function test(): Promise<void> {
     if (result.error) {
       process.exit(1);
     }
-
```

Comment:
This used to run the suites from here, now it just performs preparation for suites and they're actually run from elsewhere